templates/common/infra/terraform/applicationinsights.tf

```diff
@@ -0,0 +1,1241 @@
+terraform {

```

Comment:
Feels like all these `appinsights` files should be in an `appinsights` folder and treated like an actual TF module?

Reply:
moved to a sub folder under modules